### ISSUE TITLE
Split Lens reader out to its own repository

### ISSUE THREAD

**Issue by CAYdenberg** [codeforscience/sciencefair#136]
Subtask of #118. Ideally use [Lens-starter](https://github.com/elifesciences/lens-starter) to add custom panels and converters. The built project could then be included in ScienceFair as an npm module.

**Comment by blahah** [comment]
@CAYdenberg this work was originally done in https://github.com/blahah/lens-starter

I've imported that fork to a new repo [here](https://github.com/sciencefair-land/lens-builder), where we can expand the tooling to build and publish the compiled lens reader.

I will merge all the changes I made to the compiled Lens back into the builder.

**Comment by CAYdenberg** [comment]
OK, I see. So running `gulp` in that project should build a `dist` directory that has everything you need in it? Or did you do it some other way (I notice there's check-in `lens` folder in that project but the gulp destinations are still `dist`)

**Comment by blahah** [comment]
@CAYdenberg yeah that `lens` dir is still hanging around from the olden days. I'll clean up the repo so it has a reasonable structure and workflow, then we can start planning where it goes next.

I think at least for now it needs:

1. to build a distributable lens bundle
2. to be able to release just that distributable bundle as `sciencefair-lens` or something that we can depend on
